test-dsl/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.smartbear.readyapi</groupId>
+    <artifactId>test-dsl</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+
+    <properties>
+        <!-- Groovy version needs to be kept in sync with the Groovy Compiler Maven Plugins -->
+        <groovy.version>2.4.4</groovy.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.smartbear.readyapi</groupId>
+            <artifactId>readyapi4j-testserver-client</artifactId>
```

Comment:
In that case we'll have a transitive dependency on the SoapUI core, which I think would be unfortunate. We need to do something smarter so that you can use just the testserver client if you want to. But I'll save that one for later.